app/services/idv/session.rb

```diff
@@ -19,12 +19,13 @@ class Session
       resolution_successful
     ].freeze
 
-    attr_reader :current_user, :gpo_otp, :issuer
+    attr_reader :current_user, :gpo_otp, :service_provider, :issuer
 
-    def initialize(user_session:, current_user:, issuer:)
+    def initialize(user_session:, current_user:, service_provider:)
       @user_session = user_session
       @current_user = current_user
-      @issuer = issuer
+      @service_provider = service_provider
+      @issuer = service_provider&.issuer
```

Comment:
I'm not seeing where `@issuer` is used in this class, I think we can remove this line?

```suggestion
```